products/sle/main.pm

```diff
@@ -922,6 +922,26 @@ sub load_feature_tests() {
     loadtest "feature/feature_console/suseconnect.pm";
 }
 
+sub load_online_migration_tests() {
+    # stop packagekit service and more
+    loadtest "online_migration/sle12_online_migration/online_migration_setup.pm";
+    loadtest "online_migration/sle12_online_migration/register_system.pm";
```

Comment:
But that way I how do you actually test the addons and modules are working before upgrading? Just installing them and assuming they're okay? I don't like the sound of that.